packages/create-block-app/scripts/create-block-app.js

```diff
@@ -0,0 +1,56 @@
+const { promisify } = require("util");
+const child_process = require("child_process");
+const fs = require("fs");
+const path = require("path");
+
+const exec = promisify(child_process.exec);
+
+(async () => {
+  const blockName = process.argv[2];
+  const blockPath = process.argv[3];
```

Comment:
I think it's worth making the path optional, and just defaulting to using a directory with the same name as the `blockName` (similar to how `create-react-app` automatically creates a directory with your react app's name). Happy for this to be done in follow-up though.

Reply:
Yeah it make sense to have block path equal to block name by default. The problem with doing so now is that blocks will be created _inside_ the repo, which may lead to misinterpretation. We want third-party blocks to be stored in external repos rather than ours, but may end up seeing PRs with new blocks.

When we can run `npx create block-app myblock`, omitting path will make more sense because the command will be probably called not from the repo folder.